src/pybind/mgr/ansible_orchestrator/module.py

```diff
@@ -0,0 +1,364 @@
+"""
+ceph-mgr Ansible orchestrator module
+
+The external Orchestrator is the Ansible runner service (RESTful https service)
+"""
+
+# Python stuff
+from threading import Event
+import errno
+import json
+
+# Ceph stuff
+from mgr_module import MgrModule
+import orchestrator
+
+# Orchestrator stuff
+# A Client is used to communicate with the Ansible Runner service
+from ansible_runner_svc import Client, PlayBookExecution, ExecutionStatusCode,\
+                               EVENT_DATA_URL
+
+# Constants section
+
+# Time to clean the completions list
+WAIT_PERIOD = 10
+
+
+# List of playbooks names used
+GET_INVENTORY_PLAYBOOK = "probe-disks.yml"
```

Comment:
Also the name is weird, you call it `GET_INVENTORY_PLAYBOOK` but is this the playbook or the inventory? It's confusing. Based on https://github.com/ceph/ceph/pull/24445/files#diff-5940840b32ed5f2781084ee6e9a7d408R270 we would think it's the playbook but https://github.com/ceph/ceph/pull/24445/files#diff-5940840b32ed5f2781084ee6e9a7d408R261 indicates an inventory...

Reply:
This constant  must contain the name of the playbook used to retrieve the list of storage devices present in the host affected by the playbook run.

As you pointed the name of the playbook is weird... 
i will change it. I think that "get_storage_devices.yml" is more understandable.

Reply:
Finally changed to "host-disks.yml".